### DISCUSSION TITLE
Discord Embed

### DISCUSSION THREAD

**Discussion by Coldmints** [Hubro/clipface#58] [Q&A] [answered]
How do I get my clips to embed on Discord? I'm a super noob, some help would be appreciated. 
Thank you!

![image](https://user-images.githubusercontent.com/105855604/169271932-10271eef-bfb7-4dc9-98f5-22a51070f613.png)

**Comment by Hubro** [comment, marked as answer]
It must be an URL that the Discord servers can reach. In other words, you can't link to "localhost".

Host your Clipface instance on a public IP address, then paste the URL just like you did.

**Reply by Coldmints** [reply]
Could you give me an example of that?

**Reply by Hubro** [reply]
That's a very basic computer networking question and out of scope for Clipface. I don't have time to provide that kind of support, unfortunately. I recommend opening a question on [Server Fault](https://serverfault.com) or [Super User](https://superuser.com).
